src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
+const daysWithLowADiscount = 3;
+const daysWithLargeADiscount = 7;
+const rentOfEveryDay = 40;
+const lowDiscount = 20;
+const largeDiscount = 50;
+
 function calculateRentalCost(days) {
```

Comment:
The function `calculateRentalCost` is appropriately named and describes its purpose well.